x-pack/plugin/ent-search/build.gradle

```diff
@@ -38,6 +38,13 @@ dependencies {
   module ':modules:search-business-rules'
 }
 
+testClusters.configureEach {
+  testDistribution = 'DEFAULT'
```

Comment:
@breskeby we should think about making it simpler to setup clusters that require security w/o having to use the default distribution. I suspect the vast majority of tests that use the default distro is for this reason alone.

Otherwise these changes LGTM.